leveldbjni/src/main/native-package/src/config.h.in

```diff
@@ -36,8 +36,7 @@
 /* Define to 1 if you have the <unistd.h> header file. */
 #undef HAVE_UNISTD_H
 
-/* Define to the sub-directory in which libtool stores uninstalled libraries.
-   */
+/* Define to the sub-directory where libtool stores uninstalled libraries. */
```

Comment:
I don't think we need this change?

Reply:
https://github.com/utsaslab/leveldbjni/pull/1/files/4ece7fed84bed249e85ecc72d6f4e81d68a883b9#r219715836